.github/workflows/pr-check.yaml

```diff
@@ -84,3 +84,87 @@ jobs:
 
       - name: Run build
         run: yarn build
+  
+  e2e-tests:
+    name: bootc e2e tests
+    runs-on: ubuntu-22.04
+    env:
+      TESTING_ON_PRCHECK: true
```

Comment:
Suggestion: you could use GITHUB_ACTIONS env. var, and/or GITHUB_WORKFLOW which has the name of the workflow to get information from where the test is being run.

Reply:
my suggestion here would be to avoid to say how it's run but what action is skipped or done

like SKIP_UNINSTALL, SKIP_SETUP or SKIP_LIFECYCLE so you're not linking the github action to the test

because I could use the same setup by trying to run tests in development mode (and I'm not at this point with a PR check)

Reply:
nice idea, I've switched the environment variable to be named SKIP_INSTALLATION